README.md

```diff
@@ -17,15 +17,15 @@
 
 The e18e (Ecosystem Performance) project is an initiative to bring together the groups and individuals who are passionate about improving performance of the JavaScript ecosystem.
 
-Many ongoing efforts are already happening in this space, from [dependency tree cleanups](https://github.com/43081j/ecosystem-cleanup) to [performance optimizations](https://marvinh.dev/blog/speeding-up-javascript-ecosystem/), and much more.
+Many ongoing efforts are already happening in this space, from [dependency tree cleanups](https://github.com/43081j/ecosystem-cleanup) to [performance optimizations](https://marvinh.dev/blog/speeding-up-javascript-ecosystem/) and much more.
 
-Our aim is to provide a space for contributions, ideas and knowledge sharing around the importance of performance across the ecosystem. Hopefully, with this collaboration, we can improve visibility of the impact of things like dependency choices. Join us at the [e18e Discord Server](https://chat.e18e.dev) to connect with other like-minded devs.
+Our aim is to provide a space for contributions, ideas and knowledge sharing around the importance of performance across the ecosystem. Hopefully, with this collaboration, we can improve visibility of the impact of things like dependency choices. Join us at the [e18e Discord server](https://chat.e18e.dev) to connect with other like-minded devs.
 
 Currently, there are three main areas of focus:
 
-- [cleanup](https://e18e.dev/guide/cleanup) - cleaning up dependency trees and modernizing popular tools and libraries across the ecosystem.
-- [speedup](https://e18e.dev/guide/speedup) - speeding up parts of the ecosystem many of us depend on.
-- [levelup](https://e18e.dev/guide/levelup) - documenting and providing modern, lighter alternatives to established tools and libraries we all regularly use.
+- [Cleanup](https://e18e.dev/guide/cleanup): cleaning up dependency trees and modernizing popular tools and libraries across the ecosystem.
+- [Speedup](https://e18e.dev/guide/speedup): speeding up parts of the ecosystem many of us depend on.
+- [Levelup](https://e18e.dev/guide/levelup): documenting and providing modern, lighter alternatives to established tools and libraries we all regularly use.
```

Comment:
should these ones be lowercase? as "brand" names?

Reply:
I personally think it's ok. It's like webpack but people call it Webpack all the time, so might as well embrace it 😄